manifests/repo/apt.pp

```diff
@@ -41,16 +41,18 @@
 
   # Key has changed since 3.9
   $repo_key_name = $release ? {
-    /(3.0|3.1|3.2|3.3|3.4|3.5|3.6|3.7|3.8)/ => 'A4703C37D3F4DE7F1819E980FE79BB52D5DC52DC',
-    default                                 => '849512C2CA648EF425048F55C883F50CB2289A17',
+    '3.8'                                     => 'A4703C37D3F4DE7F1819E980FE79BB52D5DC52DC',
```

Comment:
I guess the alignment could be simplified here.

Reply:
Shouldn't the literal dots be escaped?

Reply:
not in this case. it's string comparsion, not a regular expression.

Reply:
Dur... totally didn't notice that part switched.